src/app/portal/features/site-header/site-header.component.html

```diff
@@ -15,28 +15,46 @@
       >
     </div>
 
-    <mat-nav-list class="basis-1/2 text-center">
+    <mat-nav-list class="my-8 basis-1/2 text-center">
       <a
-        [id]="route === '/' ? 'activePage' : ''"
-        mat-flat-button
+        [id]="route !== '/browse' ? 'activePage' : ''"
+        mat-button
+        disableRipple="true"
```

Comment:
It's to make behaviour consistent across links, since there is no rippling added to the links that try to emulate the buttons that did not use the `mat-button` feature.